src/Magnum/ImGuiIntegration/Context.h

```diff
@@ -251,6 +251,35 @@ If you don't do that, the fonts stay at the original scale, not matching the
 new UI scaling anymore. If you didn't supply any custom font, the function will
 reconfigure the builtin font automatically.
 
+@section ImGuiIntegration-Context-large-meshes Large meshes
+
+Complex user interfaces or widgets like [ImPlot](https://github.com/epezent/implot)
+may end up creating large meshes with more than 64,000 vertices. Because ImGui
+defaults to 16-bit index buffers this can lead to asserts or visual errors.
+
+If the underlying GL context supports @ref GL::Mesh::setBaseVertex() "setting the base vertex for indexed meshes"
+the rendering backend sets the @cpp ImGuiBackendFlags_RendererHasVtxOffset @ce
+flag. This lets ImGui know the backend can handle per-draw vertex offsets,
+allowing meshes with a little over 2 million vertices. Support for that
```

Comment:
Where does 2 million come from? Is this some ImGui limitation? IIUC, the vertex offset is a signed int, so 2 *billion* should be theoretically possible ... but I'd just not mention this at all, doesn't seem that important.

```suggestion
removing the 65k limitation altogether. Support for that
```

Reply:
Math is not my strong suit 🤡